src/server/heart-beat-db.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const MongoClient = require('mongodb').MongoClient;
+const url = 'mongodb://localhost:27017/heartbeat';
```

Comment:
would be cool if the url is set from environment variable or from some config file